src/main/java/growthcraft/milk/common/tileentity/TileEntityButterChurn.java

```diff
@@ -27,26 +28,26 @@
     public static enum WorkState {
         NONE,
         CHURN,
-        PRODUCE;
+        PRODUCE
     }
 
     private static AccesibleSlots accessibleSlots = new AccesibleSlots(new int[][]{
             {0},
-            {},
+            {0},
```

Comment:
This was intentional, I believe this was the top face, the slots should be inaccessible from the top slot, since that's where the shaft is used to operate it.

Reply:
ok i thought it was a typo, i'll change that and add a comment